eigen: don't try building tests

While configuring, if the tests are enabled, it checks some machine specific
stuff which we don't want to do while cross-compiling.

Upstream-Status: Inappropriate [embedded specific]

Signed-off-by: Kartik Mohta <kartikmohta@gmail.com>

--- a/CMakeLists.txt~	2012-07-02 20:33:20.330773075 -0400
+++ b/CMakeLists.txt	2012-07-02 20:34:23.515778505 -0400
@@ -307,24 +307,24 @@
 
 add_subdirectory(doc EXCLUDE_FROM_ALL)
 
-include(EigenConfigureTesting)
+#include(EigenConfigureTesting)
 # fixme, not sure this line is still needed:
-enable_testing() # must be called from the root CMakeLists, see man page
+#enable_testing() # must be called from the root CMakeLists, see man page
 
 
-if(EIGEN_LEAVE_TEST_IN_ALL_TARGET)
-  add_subdirectory(test) # can't do EXCLUDE_FROM_ALL here, breaks CTest
-else()
-  add_subdirectory(test EXCLUDE_FROM_ALL)
-endif()
-
-if(EIGEN_LEAVE_TEST_IN_ALL_TARGET)
-  add_subdirectory(blas)
-  add_subdirectory(lapack)
-else()
-  add_subdirectory(blas EXCLUDE_FROM_ALL)
-  add_subdirectory(lapack EXCLUDE_FROM_ALL)
-endif()
+#if(EIGEN_LEAVE_TEST_IN_ALL_TARGET)
+#  add_subdirectory(test) # can't do EXCLUDE_FROM_ALL here, breaks CTest
+#else()
+#  add_subdirectory(test EXCLUDE_FROM_ALL)
+#endif()
+
+#if(EIGEN_LEAVE_TEST_IN_ALL_TARGET)
+#  add_subdirectory(blas)
+#  add_subdirectory(lapack)
+#else()
+#  add_subdirectory(blas EXCLUDE_FROM_ALL)
+#  add_subdirectory(lapack EXCLUDE_FROM_ALL)
+#endif()
 
 add_subdirectory(unsupported)
 
@@ -342,7 +342,7 @@
   add_subdirectory(bench/spbench EXCLUDE_FROM_ALL)
 endif(NOT WIN32)
 
-ei_testing_print_summary()
+#ei_testing_print_summary()
 
 message(STATUS "")
 message(STATUS "Configured Eigen ${EIGEN_VERSION_NUMBER}")
--- a/unsupported/CMakeLists.txt~	2012-07-02 20:33:58.775385403 -0400
+++ b/unsupported/CMakeLists.txt	2012-07-02 20:34:06.495509672 -0400
@@ -1,7 +1,7 @@
 add_subdirectory(Eigen)
 add_subdirectory(doc EXCLUDE_FROM_ALL)
-if(EIGEN_LEAVE_TEST_IN_ALL_TARGET)
-  add_subdirectory(test) # can't do EXCLUDE_FROM_ALL here, breaks CTest
-else()
-  add_subdirectory(test EXCLUDE_FROM_ALL)
-endif()
+#if(EIGEN_LEAVE_TEST_IN_ALL_TARGET)
+#  add_subdirectory(test) # can't do EXCLUDE_FROM_ALL here, breaks CTest
+#else()
+#  add_subdirectory(test EXCLUDE_FROM_ALL)
+#endif()
